apollo-router/src/plugins/cache/entity.rs

```diff
@@ -1362,7 +1363,41 @@ fn extract_cache_keys(
 pub(crate) fn hash_entity_key(representation: &Value) -> String {
     // We have to hash the representation because it can contains PII
     let mut digest = Sha256::new();
-    digest.update(serde_json::to_string(&representation).unwrap().as_bytes());
```

Comment:
Is there an impact due to the extra heap allowation because of the maps being created?

Reply:
I don't create a new map anymore